frontend/src/components/elements/Alert/Alert.tsx

```diff
@@ -17,28 +17,27 @@
 
 import React, { ReactElement } from "react"
 
-import classNames from "classnames"
 import { Map as ImmutableMap } from "immutable"
 import { Alert as AlertProto } from "autogen/proto"
 import { StreamlitMarkdown } from "components/shared/StreamlitMarkdown"
+import AlertContainer, {
+  KIND,
+  KindTypeT,
+} from "components/shared/AlertContainer"
 
-import "./Alert.scss"
-import "assets/css/write.scss"
-
-// classes defined in assets/css/theme.scss
-const ALERT_CSS_CLASS = ImmutableMap({
-  [AlertProto.Format.SUCCESS]: "alert-success",
-  [AlertProto.Format.INFO]: "alert-info",
-  [AlertProto.Format.WARNING]: "alert-warning",
-  [AlertProto.Format.ERROR]: "alert-danger",
-})
-
-export function getAlertCSSClass(format: number): string {
-  const cname = ALERT_CSS_CLASS.get(format.toString())
-  if (cname) {
-    return cname
+export function getAlertKind(format: number): KindTypeT {
```

Comment:
Hm, is it possible to type this further as `getAlertKind(format: AlertProto.Format)`?

(That's not a rhetorical question, I genuinely don't know)